widgetssdk/src/main/java/com/glia/widgets/view/floatingvisitorvideoview/FloatingVisitorVideoView.java

```diff
@@ -48,19 +44,31 @@ public void onDestroy() {
     public void showVisitorVideo(VideoView newVideoView) {
         videoView = newVideoView;
         videoView.setZOrderMediaOverlay(true);
-        videoContainer.addView(videoView);
-        videoContainer.invalidate();
+        addView(videoView, 0);
     }
 
     public void hideVisitorVideo() {
+        removeView(videoView);
         releaseVideoStream();
-        videoContainer.removeAllViews();
-        videoContainer.invalidate();
+    }
+
+    public void showOnHold() {
+        pauseVideoStream();
+        onHoldOverlay.setVisibility(VISIBLE);
+    }
+
+    public void hideOnHold() {
+        resumeVideoStream();
+        onHoldOverlay.setVisibility(GONE);
+    }
+
+    public boolean hasVideo() {
+        return videoView != null;
     }
 
     private void init() {
         LayoutInflater.from(getContext()).inflate(R.layout.visitor_video_floating_view, this);
```

Comment:
I don't see a system in view and it's layout xml naming. Usually it's a good practice in xml name to have `activity_`, `view_` prefix. Is there any convention?

Reply:
If you use it as postfix you can have grouped views. I am following order that was there initially.

Could be nice to call parent group layouts layouts (containers) and well views remain views, activities activities etc.

But you can suggest changes :)

Reply:
ok let's keep it postfixed then for now, as this seems to be the convention here.

but there is still inconsistency in the class vs layout names: `FloatingVisitorVideo...` vs `visitor_video_floating..`